src/components/Checkbox/Checkbox.module.scss

```diff
@@ -153,4 +161,82 @@
       }
     }
   }
+
+  &--special-background-color {
+    &--color-1 {
+      input {
+        &:checked,
+        &:indeterminate {
+          background-color: #5d92fc;
```

Comment:
wszystkie kolory przenieś do zmiennych, jeśli już sa zdefiniowanie to skorzystaj z tych istniejących